### PR TITLE
Update CMakeLists.txt and README.md

### PR DESCRIPTION
**Pull request by konstantin1722**
In the original CMakeLists.txt file, the target_include_directories() command was pointing to a non-existent directory /include/libevdev-1.0/. This caused the compiler to fail to find the necessary libevdev header files, resulting in a compilation error.

To fix this error, I updated the target_include_directories() command to point to the correct location of the libevdev header files on Arch Linux, which is /usr/include/libevdev-1.0/. This ensures that the necessary header files are included during compilation.

However, hard-coding the path to the libevdev header files is not always the best solution, as the location of the header files can vary depending on the distribution and version of Linux. To make the CMakeLists.txt file more portable, I suggest using the pkg-config tool to dynamically locate libevdev header files.

In the updated CMakeLists.txt file, I added the find_package() and pkg_check_modules() commands to find and link the libevdev library using pkg-config. This ensures that the necessary header files and libraries are found and included dynamically, regardless of the location of the header files on the system.

Overall, these changes make the CMakeLists.txt file more portable and easier to use on different Linux distributions.

Removed MakeFile, it's not clear why he was here.

### CODE REVIEW

**Comment by yanjs** [comment]
Thank you for your contribution.

### About `CMakeLists.txt`

I decide to keep your change to `CMakeLists.txt`.

### About `README.md`

I'm sorry that I forgot to mention the `cmake` build commands in `README.md`. The commands should be 
```
cmake -S . -B ./build
cmake --build ./build
```

If possible, can you add these commands to `README.md` as alternative build commands instead of removing the original ones, so that the intermediate files are not tracked by `git`.

Also, I have mixed up the build and run commands in this file, which is not clear. Feel free to change them. Thanks.

### About `Makefile`

I would like to keep `make` as a build tool. Can you keep this file?

### Reply

> To fix this error, I updated the target_include_directories() command to point to the correct location of the libevdev header files on Arch Linux, which is /usr/include/libevdev-1.0/. This ensures that the necessary header files are included during compilation.

I'm also using Arch Linux. There is a symbolic link `/include` which points to `/usr/include` after the official Arch Linux installation. I'm not sure why this directory is not on your machine.

> However, hard-coding the path to the libevdev header files is not always the best solution, as the location of the header files can vary depending on the distribution and version of Linux. To make the CMakeLists.txt file more portable, I suggest using the pkg-config tool to dynamically locate libevdev header files.
>
> In the updated CMakeLists.txt file, I added the find_package() and pkg_check_modules() commands to find and link the libevdev library using pkg-config. This ensures that the necessary header files and libraries are found and included dynamically, regardless of the location of the header files on the system.
> 
> Overall, these changes make the CMakeLists.txt file more portable and easier to use on different Linux distributions.

I agree.

> Removed MakeFile, it's not clear why he was here.

This `Makefile` is not generated by `cmake`.

I originally decided to write a cross platform driver so I used `cmake`. Somehow I changed my mind and decided to focus on Linux only. To reduce the number of dependencies, I wrote a `Makefile` since `make` is more common on different Linux platforms. On Arch Linux, `make` is required by the `base-devel` package group, but `cmake` is not. I still keep the original `CMakeFiles.txt` so that you can use either `cmake` or `make` to build the program.

**Comment by konstantin1722** [comment]
> If possible, can you add these commands to `README.md` as alternative build commands instead of removing the original ones, so that the intermediate files are not tracked by `git`.

Sorry, I didn't really understand what you meant. I'm going to return your MakeFile and add the build commands you suggested to CMake, as well as separate the launch item separately in the README file. If this is correct, then let me know and I will commit.

Look so normal?
https://github.com/konstantin1722/T503d/tree/test

**Comment by yanjs** [comment]
Thanks. LGTM.

**Comment by konstantin1722** [comment]
Great. You can accept requests.

File: README.md [hunk 1]
```diff
@@ -12,13 +12,27 @@ libusb 1.0.26
...
+
 ## Usage
 
-Make and running in the foreground (administrator privilege required).
+Running in the foreground (administrator privilege required).
 
 ```console
-$ make
-# build/T503d
+sudo ./build/t503d
```
Comment (yanjs) [inline README.md]: There is a typo in this line. Can you fix it? "t503d" -> "T503d"

Comment (konstantin1722) [reply README.md]: Corrected.

**Comment by yanjs** [comment]
Merged. Thank you for your contribution.

**Pull request merged**